matminer/featurizers/tests/test_site.py

```diff
@@ -96,15 +96,15 @@ def test_op_site_fingerprint(self):
             self.assertEqual(l[i], t[i])
         ops = opsf.featurize(self.sc, 0)
         self.assertEqual(len(ops), 35)
-        self.assertAlmostEqual(int(1000 * ops[opsf.feature_labels().index(
-            'oct CN_6')]), 999)
+        self.assertAlmostEqual(round(ops[opsf.feature_labels().index(
```

Comment:
Why do you need to round if you have assertAlmostEqual() ?

If you are looking for a value very close to 1 (like 0.99), you should just use assertAlmostEqual() and not round (use the places kwarg if you need to set the number of decimal places)

If you are looking for anything that rounds to 1 (i.e., >0.5), then you should assert that the round exactly equals 1. (self.assertEqual() not self.assertAlmostEqual())

Reply:
@computron, thanks for catching that. Will change it shortly.